source/installation.html.haml

```diff
@@ -100,11 +99,13 @@ title: Installation
       %tr
         %td(colspan="2")
           %h3
+            %i.fa.fa-floppy-o
             Other platforms
 
       = package_row 'FreeBSD (Freshports)', 'https://www.freshports.org/multimedia/mpv'
-      = package_row 'NetBSD', 'http://cvsweb.netbsd.org/bsdweb.cgi/pkgsrc/multimedia/mpv/'
+      = package_row 'NetBSD', 'https://ftp.netbsd.org/pub/pkgsrc/current/pkgsrc/multimedia/mpv/index.html'
       = package_row 'OpenBSD', 'https://cvsweb.openbsd.org/cgi-bin/cvsweb/ports/multimedia/mpv/'
+      = package_row 'Haiku', 'https://depot.haiku-os.org/mpv'
```

Comment:
I don't understand the purpose of this section. I think it meant to be for
> Distributions usually package outdated, unmaintained, and unsupported versions of mpv.
so listing first-party maintained packages, seems redundant. I mean we cannot list everything.

Also on that note would be useful to list Fedora multimedia packages that are not in the main repo and fixes mpv on this distro.

Reply:
The section is called "Other" because BSDs are not Linux.
I was just looking for something else to add here so the BSDs aren't so alone :wink: 

> Also on that note would be useful to list Fedora multimedia packages that are not in the main repo

sure. I'll check.

Reply:
> The section is called "Other" because BSDs are not Linux.
> I was just looking for something else to add here so the BSDs aren't so alone 😉

I get it, but I think this is official port for Haiku and we list mostly unofficial ones when necessary. shrug